graylog2-web-interface/src/components/common/Spinner.tsx

```diff
@@ -38,7 +38,7 @@ type Props = {
  */
 const Spinner = ({ name, text, delay, ...rest }: Props) => (
   <Delayed delay={delay}>
-    <StyledIcon {...rest} name={name} spin />{text}
+    <StyledIcon {...rest} name={name} $displayMargin={!!text} spin />{text}
```

Comment:
Does it make sense to check if `text` is blank as well?

Reply:
Can you describe the context a bit more, what do you mean with blank?

Reply:
When for some reason `text?.trim() === ''`. Not a case that should necessarily happen, but we could catch it here just in case. It would not be visible, but maybe it could lead to unwantend indentation of the next element?

Reply:
Now I get it. I think it is a good idea, when someone wants the indentation they should style the icon instead.